WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1011,15 +1019,19 @@ extension NotificationsViewController: WPTableViewHandlerDelegate {
 //
 private extension NotificationsViewController {
     func leadingButtons(note: Notification) -> [MGSwipeButton] {
-        guard !note.read else {
-            return []
-        }
+        let isRead = note.read
+
```

Comment:
What do you think if `Notification` can have a readonly property named _isRead_?In that way we can use `notice.isRead` to read the value without updating the data model.

Reply:
I think that could make sense, although I only created this local variable because I thought it was slightly more readable in this case. I'll leave it for this PR, but it's something we could consider if we need it elsewhere :) Thanks!